### ISSUE TITLE
Fixed configuration of shared library

### ISSUE THREAD

**Issue by dtecta** [kmammou/v-hacd#10]
In src/VHACD_Lib/CMakeLists.txt, The LIB_TYPE variable with should be used for configuring a shared library (DLL) is not used in add_library(vhacd ...
I've attached a fixed CMakeLists.txt
[CMakeLists.txt](https://github.com/kmammou/v-hacd/files/140376/CMakeLists.txt)

Also it would be nice to add something similar to

set(CMAKE_DEBUG_POSTFIX  "_d")

at the top of the root CMakeLists.txt. This will add a postfix to debug binaries, which allows them to be in the same directory as their release counterparts.

Gino

**Comment by kklouzal** [comment]
Yes, currently using `SHARED` for `LIB_TYPE` will generate project files that output a .dll but no .lib file. Likewise, regardless of type, there is no distinction between release/debug binaries.
